microsoft/botframework-solutions
# [Enterprise Bot Generator][TypeScript] Add tests to the Generator

## Description
- Add tests to the generator
- Delete `yo-rc` file

## Related Issue
Fix [#614](https://github.com/Microsoft/AI/issues/614) 

## Testing Steps
1. Open `AI\templates\Enterprise-Template\src\typescript\generator-botbuilder-enterprise`
2. Run `npm i`
3. Run `npm run test` and check the running tests.

![image](https://user-images.githubusercontent.com/11904023/51695878-dd74cc80-1fe2-11e9-906a-a4ce86806799.png)


## Checklist
~- [ ] I have commented my code, particularly in hard-to-understand areas~

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant
~- [ ] A duplicate issue is filed to track future  work.~

If you have updated responses or `.lu` files:
~- [ ] All languages have been updated~
~- [ ] You have tested deployment with your new models~
